seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -0,0 +1,90 @@
+import mock
+import responses
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from seqr.views.utils.terra_api_utils import get_anvil_billing_projects, get_anvil_profile, has_anvil_session,\
+    list_anvil_workspaces, get_anvil_workspace_acl
+from settings import TERRA_API_ROOT_URL
+
+
+class TerraApiUtilsCase(TestCase):
+    fixtures = ['users']
+
+    def setUp(self):
+        google_url = 'https://oauth2.googleapis.com/token'
+        content = b'{"access_token":"ya29.c.KpMB4QdMNGgxWQvj3PxOgbQqfWQznoHuuoY03vN_A-W_W3vkdXJ6q-a539jSZEWc602cBL_RfNUH4GZV-d8rosfzQwBHoCVxfQs-Su2jBgKEdTui53jfjyU9T9LxshDl8ov31yk8Pi8f9D9ER7EUp-W1mP9gSy9e4YWkLb_q4hdJRR3BJywlQIWvPEfawpjC5Jpt_qGJ","expires_in":3599,"token_type":"Bearer"}'
+        responses.add(responses.POST, google_url, status = 200, body = content)
+        self.user = User.objects.get(email__iexact = 'test_user@test.com')
+        has_anvil_session(self.user)
```

Comment:
`has_anvil_session` sounds like something that just checks whether or not the user has a session and returns a boolean. What good does calling it here do? If its actually setting up a session you need to rename it

Reply:
you also need to test the cases where the user does not have permission to access terra

Reply:
It must be `get_session()` which will create a session. I will update it.